CMakeLists.txt

```
@@ -226,7 +226,7 @@ else ()
              -Wsuggest-override \
              -Wunused-macros \
              -Wzero-as-null-pointer-constant")
-    if(Clang STREQUAL ${CMAKE_CXX_COMPILER_ID})
+    if(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
```

```suggestion
    if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
```

Better to quote the variable since [CMake if command expands unquoted string if it happens to be a variable name](https://cmake.org/cmake/help/latest/command/if.html#variable-expansion).

---

Huh, surprisingly only today that I learned this. Thanks!